### PR TITLE
Update Elegy for JAX description

### PR DESCRIPTION
**Pull request by 8bitmp3**
Updating the description of Elegy.

Thanks @cgarciae for the awesome work on Elegy (it's like Keras + PyTorch Lightning with Flax, Haiku, Optax, TensorFlow Datasets, and Pytorch DataLoaders support).

### CODE REVIEW

**Comment by cgarciae** [comment]
Maybe something simpler like
```
A framework-agnostic Trainer interface for the Jax ecosystem. Supports Flax, Haiku, and Optax.
```
would do the job 😄

File: readme.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ This is a curated list of awesome JAX libraries, projects, and other resources.
     - [Flax](https://github.com/google/flax) - Centered on flexibility and clarity. <img src="https://img.shields.io/github/stars/google/flax?style=social" align="center">
     - [Haiku](https://github.com/deepmind/dm-haiku) - Focused on simplicity, created by the authors of Sonnet at DeepMind. <img src="https://img.shields.io/github/stars/deepmind/dm-haiku?style=social" align="center">
     - [Objax](https://github.com/google/objax) - Has an object oriented design similar to PyTorch. <img src="https://img.shields.io/github/stars/google/objax?style=social" align="center">
-    - [Elegy](https://poets-ai.github.io/elegy/) - Implements the Keras API with some improvements. <img src="https://img.shields.io/github/stars/poets-ai/elegy?style=social" align="center">
+    - [Elegy](https://poets-ai.github.io/elegy/) - Provides the Keras-like high-level API and functional PyTorch Lightning-like low-level API; supports Flax, Haiku, Optax, as well as TensorFlow Datasets and Pytorch DataLoaders; is 100% framework-agnostic on the low-level API. <img src="https://img.shields.io/github/stars/poets-ai/elegy?style=social" align="center">
```
Comment (n2cholas) [inline readme.md:24]: As per @cgarciae's suggestion, I also prefer

```
A framework-agnostic Trainer interface for the JAX ecosystem. Supports Flax, Haiku, and Optax.
```

**Comment by n2cholas** [comment]
Sorry for the late reply on this @8bitmp3, thanks for making this update. I also like @cgarciae suggestion--it's concise and to the point. What do you think @8bitmp3?

**Comment by 8bitmp3** [comment]
Done @n2cholas 💰

**Comment by n2cholas** [comment]
Thank you @8bitmp3!

**Pull request merged**